operator/v1/register.go

```diff
@@ -38,6 +38,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&KubeControllerManagerList{},
 		&OpenShiftAPIServer{},
 		&OpenShiftAPIServerList{},
+		&OpenShiftControllerManagerConfig{},
```

Comment:
I don't think Config adds anything here.  It's in the config group.  The master team has chosen to omit it because the name describes the object that is being managed.  For consistency alone, we'd choose to omit name.

Reply:
> I don't think Config adds anything here. It's in the config group. 

actually it is in the "operator" group.  Nothing about the api group name says "config" to an end user, nor is there any guarantee that this group remains config objects only in the future.

> For consistency alone, we'd choose to omit Config.

just because someone else got there first w/ the wrong answer doesn't mean everyone else should copy the wrong answer.

Reply:
(i've said my piece, so i'll update this as requested and move on as usual).